apis_configs/fence_settings.py

```diff
@@ -124,6 +124,12 @@ def get_from_dict(dictionary, key, default=''):
     GOOGLE_GROUP_PREFIX = get_from_dict(data, 'GOOGLE_GROUP_PREFIX', 'gen3')
     SUPPORT_EMAIL_FOR_ERRORS = get_from_dict(
         data, 'SUPPORT_EMAIL_FOR_ERRORS', None)
+    WHITE_LISTED_SERVICE_ACCOUNT_EMAILS = (
+        get_from_dict(data, 'WHITE_LISTED_SERVICE_ACCOUNT_EMAILS', [])
+    )
+    WHITE_LISTED_GOOGLE_PARENT_ORGS = (
+        get_from_dict(data, 'WHITE_LISTED_GOOGLE_PARENT_ORGS', [])
+    )
 
```

Comment:
The code in fence that looks for these new variables will behave gracefully if they're not there?  If yes, then I'll approve! :+1: